docs/api/page.rst

```diff
@@ -579,6 +526,69 @@ In a nutshell, this is what you can do with MuPDF.NET:
       :rtype: IEnumerable<Annot>
       :returns: an :ref:`Annot` for each iteration.
 
+   .. method:: GetBboxlog(bool layer: false)
+
+      :returns: a list of BoxLog that envelop text, image or drawing objects. Each item is a tuple `(type, (x0, y0, x1, y1))` where the second tuple consists of rectangle coordinates, and *type* is one of the following values. If `layers = true`, there is a third item containing the OCG name or `null`: `(type, (x0, y0, x1, y1), null)`.
+
+         * `"fill-text"` -- normal text (painted without character borders)
+         * `"stroke-text"` -- text showing character borders only
+         * `"ignore-text"` -- text that should not be displayed (e.g. as used by OCR text layers)
+         * `"fill-path"` -- drawing with fill color (and no border)
+         * `"stroke-path"` -- drawing with border (and no fill color)
+         * `"fill-image"` -- displays an image
+         * `"fill-shade"` -- display a shading
+
+         The item sequence represents the **sequence in which these commands are executed** to build the page's appearance. Therefore, if an item's bbox intersects or contains that of a previous item, then the previous item may be (partially) covered / hidden.
+
+         So this list can be used to detect such situations. An item's index in this list equals the value of a `"SeqNo"` in dictionaries as returned by :meth:`Page.GetDrawings` and :meth:`Page.GetTexttrace`.
```

Comment:
The method should be `Page.GetTextTrace` so as to correctly reference the method in the docs.